BedrockServer.cpp

```diff
@@ -1385,6 +1427,12 @@ void BedrockServer::_status(BedrockCommand& command) {
         content["version"]  = _version;
         content["host"]     = _args["-nodeHost"];
 
+        {
+            // Make it known if anything is known to cause crashes.
+            shared_lock<decltype(_crashCommandMutex)> lock(_crashCommandMutex);
+            content["crashCommands"] = _crashCommands.size();
```

Comment:
can we make this a list of commands instead of the number of commands?

Reply:
Yes, we could, but it's not very straightforward, because a "crash command" is a combination of a methodLine and a set of name/value pairs, which isn't going to be super obvious to look at.